### PR TITLE
add parental view

### PR DESCRIPTION
**Pull request by tiemfah**
-  [x] new field in CustomUser table: parental_key (uuid4)
-  [x] user can access other people profiles with their parental key

### CODE REVIEW

**Comment by chiiida** [comment]
I can not click on `request access` button and go through it. Did I have to do something else other than migrate?

File: templates/request_user.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html' %} {% block content %}
+
+<h1>Parental Access</h1>
+<p>please enter your child key to view thier profile.</p>
+<form method="post">
+  {% csrf_token %}
+  <input type="text" name="uuid" id="uuid">
+  
+</form>
+<button class="" type="submit" formmethod="post">Request access</button>
```
Comment (chiiida) [inline templates/request_user.html]: I think button should be inside form tag.

**Review by chiiida — CHANGES_REQUESTED** [review]

File: templates/request_user.html (hunk 1 above)
Comment (tiemfah) [reply templates/request_user.html]: fixed

**Review by chiiida — APPROVED** [review]
It looks good now. 👍

**Review by Sirikonss — APPROVED** [review]

**Pull request merged**